symbolic-symcache/tests/test_writer.rs

```diff
@@ -268,3 +271,69 @@ fn test_transformer_symbolmap() -> Result<(), Error> {
 
     Ok(())
 }
+
+#[cfg(feature = "il2cpp")]
+#[test]
+fn test_mapless_usym() -> Result<(), Error> {
+    let buffer = ByteView::open(fixture("il2cpp/artificial.usym"))?;
```

Comment:
TIL about the Byteview::open shortcut...

Reply:
credit probably goes to jan here since the copy pasted code eventually leads back to him!